buildroot/share/tests/LPC1769_tests

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# Build tests for LPC1769 (NXP ARM Cortex-M3)
+#
+
+# exit on first failure
+set -e
+
+restore_configs
+use_example_configs Azteeg/X5GT
+exec_test $1 $2 "Azteeg X5GT Example Config"
+
+restore_configs
+opt_set MOTHERBOARD BOARD_SMOOTHIEBOARD
+opt_set EXTRUDERS 2
+opt_set TEMP_SENSOR_1 -1
+opt_set TEMP_SENSOR_BED 5
+opt_enable REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER SDSUPPORT ADAPTIVE_FAN_SLOWING NO_FAN_SLOWING_IN_PID_TUNING \
```

Comment:
Smoothieboard only supports `VIKI2` and `miniVIKI` lcd options.

Reply:
Apparently the `DOGLCD_A0` pin is commented out. So this gives another compile error. I've asked the author of the Smoothieboard pins file to clarify at https://github.com/MarlinFirmware/Marlin/commit/a1a5367380#r32380297

Reply:
Smoothieboard should also work with the RepRapSmartDiscount flavour of LCD's as far as I understand, although I haven't tried it yet.

Reply:
We'll just have to figure out the proper pins to use. Does it share the same EXP1/2 layout as MKS SBASE, I wonder…?

Reply:
http://chibidibidiwah.wdfiles.com/local--files/panel/smoothieboard2sd.jpg
http://chibidibidiwah.wdfiles.com/local--files/panel/GLCDPINOUTS2SD.jpg

Reply:
Looks like these are the same:
  #define SCK_PIN          P0_07
  #define MISO_PIN         P0_08
  #define MOSI_PIN         P0_09

I'm not sure how these pins should be managed.
  #define SS_PIN  
  #define ONBOARD_SD_CS 
  #define SD_DETECT_PIN